scenarios/kubernetes_bazel.py

```diff
@@ -52,15 +56,58 @@ def get_version():
 
 def main(args):
     """Trigger a bazel build/test run, and upload results."""
+    # pylint:disable=too-many-branches
+    if args.install:
+        for install in args.install:
+            if not os.path.isfile(install):
+                raise ValueError('Invalid install path: %s' % install)
+            check('pip', 'install', '-r', install)
+
     check('bazel', 'clean', '--expunge')
     res = 0
     try:
-        if args.build:
+        if args.build is not None:
             check('bazel', 'build', *args.build.split(' '))
```

Comment:
build should also probably use the "affected binaries" code.

(possibly removing the `kind(.* binary)` wrapper?)